packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -95,15 +88,30 @@
                 height: 4rem;
                 width: 100%;
                 padding: 0.5rem;
-                border-radius: 1.6rem 1.6rem 0 0;
+                border-radius: 1.4rem 1.4rem 0 0;
```

Comment:
since jurisdiction banner file is deleted, pls remove the styles related to that file. (line 40)